I can't build it.

Hi,
When I try to build it I got an error about missing dependencies for MonadPlus and MonadFix.
Could you help with it ?

``` haskell
exceptiot  > /tmp/stack-a54ac3b56d3985e8/exceptiot-0.0.1.0/src/Control/Monad/Except/Catch.hs:31:27: error: [GHC-76037]
exceptiot  >     Not in scope: type constructor or class ‘MonadFix’
exceptiot  >     Suggested fix: Perhaps use ‘MonadFail’ (imported from Prelude)
exceptiot  >    |
exceptiot  > 31 |         , MonadWriter w , MonadFix, MonadFail, MonadThrow, MonadCatch, MonadMask
exceptiot  >    |                           ^^^^^^^^
exceptiot  > [2 of 3] Compiling Control.Monad.Except.IO
exceptiot  > 
exceptiot  > /tmp/stack-a54ac3b56d3985e8/exceptiot-0.0.1.0/src/Control/Monad/Except/IO.hs:31:49: error: [GHC-76037]
exceptiot  >     Not in scope: type constructor or class ‘MonadPlus’
exceptiot  >    |
exceptiot  > 31 |         ( Functor, Applicative, Monad, MonadIO, MonadPlus, Alternative
exceptiot  >    |                                                 ^^^^^^^^^
exceptiot  > 
```

--- a/.github/workflows/haskell.yml
+++ b/.github/workflows/haskell.yml
@@ -16,11 +16,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.8.4", "8.10.4", "9.0.2", "9.2.4", "9.4.2"]
+        ghc: ["8.8.4", "8.10.4", "9.0.2", "9.2.4", "9.4.2", "9.6", "9.8", "9.10"]
         experimental: [false]
     steps:
-      - uses: actions/checkout@v2
-      - uses: haskell/actions/setup@v1
+      - uses: actions/checkout@v4
+      - uses: haskell-actions/setup@v2
         id: setup-haskell-cabal
         with:
           ghc-version: ${{ matrix.ghc }}
@@ -40,8 +40,5 @@ jobs:
         continue-on-error: ${{ matrix.experimental }}
       - run: cabal v2-test --disable-optimization -j $CONFIG --test-options "--fail-on-focus"
         continue-on-error: ${{ matrix.experimental }}
-      - if: ${{ matrix.ghc != '8.6.5' }}
-        continue-on-error: ${{ matrix.experimental }}
-        run: cabal v2-haddock -j $CONFIG
       - run: cabal v2-sdist
         continue-on-error: ${{ matrix.experimental }}

--- a/changelog.md
+++ b/changelog.md
@@ -1,5 +1,11 @@
 # exceptiot
 
+## 0.0.1.1
+
+* [#]()
+    * Add imports for `Control.Monad.Fix` and `Control.Monad`, allowing to build
+      with newer `mtl`.
+
 ## 0.0.1.0
 
 * [#1](https://github.com/parsonsmatt/exceptiot/pull/1)

--- a/changelog.md
+++ b/changelog.md
@@ -2,7 +2,7 @@
 
 ## 0.0.1.1
 
-* [#]()
+* [#3](https://github.com/parsonsmatt/exceptiot/pull/3)
     * Add imports for `Control.Monad.Fix` and `Control.Monad`, allowing to build
       with newer `mtl`.
 

--- a/exceptiot.cabal
+++ b/exceptiot.cabal
@@ -1,11 +1,11 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.35.0.
+-- This file has been generated from package.yaml by hpack version 0.36.0.
 --
 -- see: https://github.com/sol/hpack
 
 name:           exceptiot
-version:        0.0.1.0
+version:        0.0.1.1
 synopsis:       ExceptT, but uses IO instead of Either
 description:    Please see the README on Github at <https://github.com/parsonsmatt/exceptiot#readme>
 category:       Control

--- a/package.yaml
+++ b/package.yaml
@@ -1,5 +1,5 @@
 name:                exceptiot
-version:             0.0.1.0
+version:             0.0.1.1
 github:              "parsonsmatt/exceptiot"
 license:             BSD3
 author:              "Matt Parsons"

--- a/src/Control/Monad/Except/Catch.hs
+++ b/src/Control/Monad/Except/Catch.hs
@@ -11,9 +11,10 @@ module Control.Monad.Except.Catch
     ) where
 
 import Control.Applicative
+import Control.Monad
 import Control.Monad.Catch
 import Control.Monad.Error.Class hiding (modifyError)
-import qualified Control.Monad.Except as Except
+import Control.Monad.Fix
 import Control.Monad.IO.Unlift
 import Control.Monad.Reader
 import Control.Monad.State
@@ -48,7 +49,7 @@ instance (MonadCatch m, Exception e) => MonadError e (ExceptCatchT e m) where
 runExceptCatchT :: (Exception e, MonadCatch m) => ExceptCatchT e m a -> m (Either e a)
 runExceptCatchT = try . unsafeRunExceptCatchT
 
--- | Like 'Except.modifyError', but it selects the 'ExceptCatchT' instance for 'IO'
+-- | Like 'Control.Monad.Except.modifyError', but it selects the 'ExceptCatchT' instance for 'IO'
 -- exceptions instead of the 'ExceptT' instance with an 'Either' error.
 --
 -- @since 0.1.0.0

--- a/src/Control/Monad/Except/IO.hs
+++ b/src/Control/Monad/Except/IO.hs
@@ -11,9 +11,11 @@ module Control.Monad.Except.IO
     ) where
 
 import Control.Applicative
+import Control.Monad
 import Control.Monad.Catch hiding (catch, try)
 import Control.Monad.Error.Class hiding (modifyError)
 import qualified Control.Monad.Except as Except
+import Control.Monad.Fix
 import Control.Monad.Reader
 import Control.Monad.State
 import Control.Monad.Writer

